azure-iot-hub-devicesdk/setup.py

```diff
@@ -40,6 +40,7 @@
         "transitions>=0.6.8,<1.0.0",
         "requests>=2.20.0,<3.0.0",
         "requests-unixsocket>=0.1.5,<1.0.0",
+        "janus>=0.4.0,<1.0.0;python_version>='3.5'",
```

Comment:
>"janus>=0.4.0,<1.0.0;python_version>='3.5'", [](start = 8, length = 44)

For my own education, I notice this is here but not in requirements.txt. It looks like requirements.txt should have dependencies that would be used by a developer and setup.py should have dependencies that are needed at runtime.  Is this right?  If so, there's a few things in requirements.txt (paho and transitions) that don't need to be there because they're in setup.py

Reply:
Wait.  I just found your requirements.txt changes.  I'm still curious about the difference because I hate that dependencies are recorded in two different places.  

---
In reply to: [264335510](https://github.com/Azure/azure-iot-sdk-python-preview/pull/55#discussion_r264335510) [](ancestors = 264335510)